Summary.md

```diff
@@ -1,5 +1,20 @@
 # Summary
 
-* [New introduction](/docs/Readme.md)
+* [Quickstart](/docs/Readme.md)
 * [Contributing]()
-    * [Docs](/docs/contributing/Docs.md)
```

Comment:
This should be `docs/contributing.md` because it describes how to contribute. It's not `installation` file. Can we change it that way?